controls/gdal_exists.rb

```diff
@@ -16,9 +16,129 @@
+  describe file(gdallocationinfo_full_path) do
+    it { should exist }
+    it { should be_executable }
+  end
+  gdalmanage_full_path = File.join(plan_installation_directory.stdout.strip, "bin/gdalmanage")
+  describe file(gdalmanage_full_path) do
+    it { should exist }
+    it { should be_executable }
+  end
+  gdalserver_full_path = File.join(plan_installation_directory.stdout.strip, "bin/gdalserver")
+  describe file(gdalserver_full_path) do
+    it { should exist }
+    it { should be_executable }
+  end
+  gdalsrsinfo_full_path = File.join(plan_installation_directory.stdout.strip, "bin/gdalsrsinfo")
+  describe file(gdalsrsinfo_full_path) do
+    it { should exist }
+    it { should be_executable }
+  end
+  gdaltindex_full_path = File.join(plan_installation_directory.stdout.strip, "bin/gdaltindex")
+  describe file(gdaltindex_full_path) do
+    it { should exist }
+    it { should be_executable }
+  end
+  gdaltransform_full_path = File.join(plan_installation_directory.stdout.strip, "bin/gdaltransform")
+  describe file(gdaltransform_full_path) do
+    it { should exist }
+    it { should be_executable }
+  end
+  gdalwarp_full_path = File.join(plan_installation_directory.stdout.strip, "bin/gdalwarp")
+  describe file(gdalwarp_full_path) do
+    it { should exist }
+    it { should be_executable }
+  end
+  gnmanalyse_full_path = File.join(plan_installation_directory.stdout.strip, "bin/gnmanalyse")
+  describe file(gnmanalyse_full_path) do
+    it { should exist }
+    it { should be_executable }
+  end
+  gnmmanage_full_path = File.join(plan_installation_directory.stdout.strip, "bin/gnmmanage")
+  describe file(gnmmanage_full_path) do
+    it { should exist }
+    it { should be_executable }
+  end
+  nearblack_full_path = File.join(plan_installation_directory.stdout.strip, "bin/nearblack")
+  describe file(nearblack_full_path) do
+    it { should exist }
+    it { should be_executable }
+  end
+  ogr2ogr_full_path = File.join(plan_installation_directory.stdout.strip, "bin/ogr2ogr")
+  describe file(ogr2ogr_full_path) do
+    it { should exist }
+    it { should be_executable }
+  end
+  ogrinfo_full_path = File.join(plan_installation_directory.stdout.strip, "bin/ogrinfo")
+  describe file(ogrinfo_full_path) do
+    it { should exist }
+    it { should be_executable }
+  end
+  ogrlineref_full_path = File.join(plan_installation_directory.stdout.strip, "bin/ogrlineref")
+  describe file(ogrlineref_full_path) do
+    it { should exist }
+    it { should be_executable }
+  end
+  ogrtindex_full_path = File.join(plan_installation_directory.stdout.strip, "bin/ogrtindex")
+  describe file(ogrtindex_full_path) do
+    it { should exist }
+    it { should be_executable }
+  end
+  testepsg_full_path = File.join(plan_installation_directory.stdout.strip, "bin/testepsg")
+  describe file(testepsg_full_path) do
+    it { should exist }
+    it { should be_executable }
```

Comment:
Could you combine these tests with a loop? An example can be viewed [here](https://github.com/chef-base-plans/python2/blob/07207f3ce5f3fe713ab59d9795358a43a65911c0/controls/python_works.rb#L60-L67)

Reply:
All 'exists' tests combined into a single array loop.